src/commands/notify.yml

```diff
@@ -24,18 +24,78 @@ parameters:
     description: Notification message
     type: string
   priority:
-    description: 'Notification priority (low-to-high: -2 to 1)'
+    description: "Notification priority (low-to-high: -2 to 1)"
     type: integer
     default: 0
+  fail_only:
+    description: If `true`, notifications successful jobs will not be sent
+    type: boolean
+    default: false
+  only_for_branches:
+    description: If set, a comma-separated list of branches for which to send notifications. No spaces.
+    type: string
+    default: ""
 
 steps:
+  - run:
+      name: Pushover - Setting Failure Condition
+      command: |
+        echo 'export PUSHOVER_BUILD_STATUS="fail"' >> $BASH_ENV
+      when: on_fail
+
+  - run:
+      name: Pushover - Setting Success Condition
+      command: |
+        echo 'export PUSHOVER_BUILD_STATUS="success"' >> $BASH_ENV
+      when: on_success
+
+  - run:
+      name: Provide error if non-bash shell
```

Comment:
I think I'd probably leave out this complexity. If `{the thing set as shell}` isn't installed, I'd expect the Circle user-experience to be good enough that we don't have to provide our own check and error message.

Reply:
I thought a little bit of that too, but this time I followed the slack-orb. I can remove it.
https://github.com/CircleCI-Public/slack-orb/blob/v3.4.2/src/commands/status.yml#L79-L85